pages/checkout.js

```diff
@@ -0,0 +1,82 @@
+const { I } = inject();
+
+module.exports = {
+  
+  firstNameField: {xpath: '//input[@name="firstname"]'},
+  lastNameField: {xpath: '//input[@name="lastname"]'},
+  addressField: {xpath: '//input[@name="address_1"]'},
+  cityField: {xpath: '//input[@name="city"]'},
+  postcodeField: {xpath: '//input[@name="postcode"]'},
+  countryToggle: {xpath: '(//div/a[@class="sbSelector"])[1]'},
+  chooseCountryLink: {xpath: '//li[.="Ukraine"]'},
+  regionToggle: {xpath: '//div/a[contains(text(), "Please Select")]'},
+  chooseRegionLink: {xpath: "//li/a[contains(text(),'Kharkivs')]"},
+  step1Toggle: {xpath: '//a[@data-toggle="collapse"]'},
+  continueButton: {xpath: '(//input[@value="Continue"])[last()]'},
+  agree: {xpath: '//input[@name="agree"]'},
+  confirmOrderButton: {xpath: '//input[@value="Confirm Order"]'},
+
+ 
+  itemPriceText: {xpath: '(//tbody/tr/td)[last()]'},
+  flatShippingRateText: {xpath: '(//tfoot/tr/td[@class="text-right"])[4]'},
+  totalPriceText: {xpath: '(//tfoot/tr/td[@class="text-right"])[6]'},
+
+  fillCheckoutForm2(checkout) {
```

Comment:
name it more specifically, like `fillBillingForm()` or `...shipping...`

Reply:
> name it more specifically, like `fillBillingForm()` or `...shipping...`

++